core/src/main/java/com/sequenceiq/cloudbreak/service/rdsconfig/ClouderaManagementServiceRdsConfigProvider.java

```diff
@@ -0,0 +1,53 @@
+package com.sequenceiq.cloudbreak.service.rdsconfig;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.database.base.DatabaseType;
+import com.sequenceiq.cloudbreak.domain.Blueprint;
+
+@Component
+public class ClouderaManagementServiceRdsConfigProvider extends AbstractRdsConfigProvider {
+
+    private static final String PILLAR_KEY = "cmmanagement";
+
+    @Value("${cb.clouderamanager.management.service.port:5432}")
```

Comment:
Suggest naming the property cb....service.database.port, since this is not the port for the service, but for its database.